### ISSUE TITLE
Unicast to single ArtNet hardware interface by IP Address

### ISSUE THREAD

**Issue by BlindElephants** [OpenLightingProject/libartnet#22]
I assume this is possible somehow but there is not clear documentation or an example that demonstrates how to create a artnet node instance and use it to send DMX messages to a single hardware interface, rather than broadcasting on the network.

**Comment by gaborpapp** [comment]
I'm encountering the same issue. Have you managed to solve this?

**Comment by BlindElephants** [comment]
Yes, we forked the repo and modified it to enable this functionality: https://github.com/MASARY-Studios/libartnet
